python_modules/libraries/dagster-looker/dagster_looker/lkml/asset_utils.py

```diff
@@ -1,11 +1,104 @@
 from pathlib import Path
-from typing import List, Sequence
+from typing import AbstractSet, Any, List, Mapping, Sequence, Tuple
 
 import lkml
 import yaml
 from dagster import AssetSpec
+from dagster._core.utils import toposort_flatten
+
+from dagster_looker.lkml.dagster_looker_lkml_translator import (
+    DagsterLookerLkmlTranslator,
+    LookMLStructureType,
+)
+
+
+def deep_merge_objs(onto_obj: Any, from_obj: Any) -> Any:
```

Comment:
Ellipsis usage in this function is... unusual (I like it!). 
Could you write a comment explaining this choice? Is this because looker can pass it to our code?